Package versions causing type errors in CI workflows

Some updated versions of packages are incompatible, causing various GH Action workflows to fail.
eg:

```
Error: ERROR in node_modules/graphql-tag/lib/index.d.ts(2,57): error TS1005: ',' expected.
Date: 2021-05-06T06:41:47.020Z
Error: node_modules/graphql-tag/lib/index.d.ts(2,63): error TS1005: ',' expected.
```

--- a/package.json
+++ b/package.json
@@ -62,7 +62,7 @@
     "diff-match-patch": "^1.0.4",
     "electron-log": "^4.2.4",
     "graphql": "^14.6.0",
-    "graphql-tag": "^2.10.0",
+    "graphql-tag": "2.11.0",
     "karma-spec-reporter": "0.0.32",
     "moment": "^2.24.0",
     "ngx-markdown": "^8.2.1",

